newIDE/app/src/Export/GenericExporters/OnlineWebExport/OnlineGameLink.js

```diff
@@ -102,7 +106,16 @@ const OnlineGameLink = ({
       const { id } = profile;
       try {
         setIsGameLoading(true);
-        const game = await getGame(getAuthorizationHeader, id, gameId);
+        const gamePromise = getGame(getAuthorizationHeader, id, gameId);
```

Comment:
I'm not sure you need to create the promise here before calling it?

Reply:
It allows to get the slug in parallel.

Reply:
if you're trying to get them in parrallel, should we use a Promise.all for it to be clearer?

```
const [slugs, game] = await Promise.all([
    getGameSlugs(...),
    getGame(...),
])
if (slugs.length > 0) {
    setSlug(slugs[0]);
}
```

Reply:
I think it's not possible because the game must be set even if there is an exception getting the slugs.

Reply:
You can `Promise.all` with a `catch(() => {console.error(...)})` on slug so that you avoid throwing if it's unsuccessful?

The risk of launching a promise and await it later is that it's possible that the promise throws in the meantime (before you `await` it) and you get a "unhandled exception error".